Untitled.py

```diff
@@ -0,0 +1,53 @@
+
+
+
+def sheet_data(sheet):
+
+    out = list()
+
+    sheetIndex = 0
```

Comment:
1) uppercase is not allowed in function variables
2) the variable's value is not used